gui/src/main/java/org/jboss/as/console/client/shared/subsys/undertow/HttpMetricPresenter.java

```diff
@@ -178,7 +179,14 @@ public void loadDetails() {
 
         ModelNode operation = new ModelNode();
         operation.get(OP).set(READ_RESOURCE_OPERATION);
-        operation.get(ADDRESS).set(Baseadress.get());
+
+        if(Baseadress.get().hasDefined("profile")) {
```

Comment:
To be honest, because I didn't know about it. Let me test using `RuntimeBaseAddress.get()`.